cdap-docs/developers-manual/source/building-blocks/datasets/system-custom.rst

```diff
@@ -101,7 +101,14 @@ To create a Dataset of type ``UniqueCountTable``, add the following into the App
 
 You can also pass ``DatasetProperties`` as a third parameter to the ``createDataset`` method.
 These properties will be used by embedded Datasets during creation and will be available via the
-``DatasetSpecification`` passed to the Dataset constructor.
+``DatasetSpecification`` passed to the Dataset constructor. For example, to create a 
```

Comment:
OrderedTable.PROPERTY_TTL. 
But I am not sure that is a good one. TTL is something that exists for streams and other datasets, too. Perhaps we should have a global one. 
Or DatasetProperties.Builder.setTTL()?